proto/buf/audit/v1/envelope.proto

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.audit.v1;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+message EventEnvelope {
+    string event_id = 1;
+    // The actor represents the subject initiating execution
+    // of the event
+    Actor actor = 2;
+    google.protobuf.Timestamp event_time = 3;
+    RequestInfo request_info = 4;
+    // Represents object(s) per event, e.g. for remote generation,
+    // both repository, template and all the template's
+    // plugins are objects.
+    repeated Object objects = 5;
```

Comment:
Why is this repeated?

Reply:
there may be a case when a single action affects more than one "object" so we need to include them all in the audit log entry